cnxdb/contrib/pytest.py

```diff
@@ -87,6 +87,12 @@ def db_init(db_engines):
     """Initializes the database"""
     from cnxdb.init.main import init_db
     init_db(db_engines['super'], is_venv_importable())
+    if is_venv_importable():
```

Comment:
I like that you only do this when the venv is active. I hadn't thought to do this here. 👍

Reply:
agreed, good catch.